packages/jitm/src/Manager.php

```diff
@@ -199,30 +229,41 @@ function jitm_enqueue_files() {
 	 * @return bool Always true
 	 */
 	function dismiss( $id, $feature_class ) {
-		JetpackTracking::record_user_event( 'jitm_dismiss_client', array(
-			'jitm_id' => $id,
-			'feature_class' => $feature_class,
-		) );
-
+		\JetpackTracking::record_user_event(
+			'jitm_dismiss_client',
+			array(
+				'jitm_id'       => $id,
+				'feature_class' => $feature_class,
+			)
+		);
 
-		$hide_jitm = Jetpack_Options::get_option( 'hide_jitm' );
+		$hide_jitm = \Jetpack_Options::get_option( 'hide_jitm' );
 		if ( ! is_array( $hide_jitm ) ) {
 			$hide_jitm = array();
 		}
 
 		if ( isset( $hide_jitm[ $feature_class ] ) ) {
 			if ( ! is_array( $hide_jitm[ $feature_class ] ) ) {
-				$hide_jitm[ $feature_class ] = array( 'last_dismissal' => 0, 'number' => 0 );
+				$hide_jitm[ $feature_class ] = array(
+					'last_dismissal' => 0,
+					'number'         => 0,
+				);
 			}
 		} else {
-			$hide_jitm[ $feature_class ] = array( 'last_dismissal' => 0, 'number' => 0 );
+			$hide_jitm[ $feature_class ] = array(
+				'last_dismissal' => 0,
+				'number'         => 0,
+			);
 		}
 
 		$number = $hide_jitm[ $feature_class ]['number'];
 
-		$hide_jitm[ $feature_class ] = array( 'last_dismissal' => time(), 'number' => $number + 1 );
+		$hide_jitm[ $feature_class ] = array(
+			'last_dismissal' => time(),
+			'number'         => $number + 1,
+		);
 
-		Jetpack_Options::update_option( 'hide_jitm', $hide_jitm );
+		\Jetpack_Options::update_option( 'hide_jitm', $hide_jitm );
```

Comment:
Should we use the options package here?

Reply:
That's the plan, just want to get what we have into features/jetpack-packages so far